PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalPaymentAuthRequest.kt

```diff
@@ -8,6 +8,8 @@ sealed class PayPalPaymentAuthRequest {
     /**
      * The request was successfully created and is ready to be launched by [PayPalLauncher]
      */
+    // TODO: requestParams should be internal but can't be until PayPalClientUnitTest is converted
+    //  to Kotlin (same for all modules with ReadyToLaunch request)
```

Comment:
Converting all of the client unit tests to Kotlin is definitely the end goal but it is a large task. Are there any other ways to mark these params as internal until we convert all the tests? Maybe doc strings would be ok for now? @tdchow

Reply:
Are we seeing compile errors caused by the internal Kotlin scopes? I was seeing these errors in Java unit tests but they were still able to compile and run.

Reply:
Actual compiler errors - "cannot find symbol `getRequestParams`"

Reply:
Gotcha, yeah I think using doc strings is a good idea.